practices/understanding_git.md

```diff
@@ -1,5 +1,5 @@
 ---
-slug: understanding_git
+slug: understanding-git
```

Comment:
```suggestion
slug: git
```

This change is something small. But it can serve as a quick introduction to slugs. Generally, in the SEO (Search Engine Optimization) community is a common practice to make slugs as short as possible.  The idea is that the short and on-point slug is better for both person and search engine (more on this here https://yoast.com/slug/). 

In this case, this great article is about Git. You don't need to add any other parts to slug :)

Reply:
okk cool